lib/libvfio-user.c

```diff
@@ -188,17 +189,163 @@ dump_buffer(const char *prefix, const char *buf, uint32_t count)
     if (i % bytes_per_line != 0) {
         fprintf(stderr, "\n");
     }
-}
-#else
-#define dump_buffer(prefix, buf, count)
 #endif
+}
 
 static bool
 is_migr_reg(vfu_ctx_t *vfu_ctx, int index)
 {
     return &vfu_ctx->reg_info[index] == vfu_ctx->migr_reg;
 }
 
+static ssize_t
+region_access(vfu_ctx_t *vfu_ctx, size_t region_index,
+              bool is_write, char *buf, uint64_t offset, size_t count)
```

Comment:
Can we swap offset and count to make it consistent with the rest of the code? (Which is think matches `pwrite(2)`.)

Reply:
good spot, done